deno.jsonc

```diff
@@ -18,11 +18,10 @@
 		"docker:build": "docker build -t opengb -f cli.Dockerfile .",
 
 		// Generates schema
-		"artifacts:build:all": "deno task format && deno task artifacts:build:schema && deno task artifacts:build:runtime_archive && deno task artifacts:build:prisma_archive && deno task artifacts:build:dynamic_archive",
+		"artifacts:build:all": "deno task format && deno task artifacts:build:schema && deno task artifacts:build:runtime_archive && deno task artifacts:build:prisma_archive",
```

Comment:
The new generated sdks use the dynamic archive pretty heavily, how should we handle this future merge conflict?